openshift/e2e-common.sh

```diff
@@ -100,18 +100,30 @@ function run_e2e_rekt_tests(){
   echo "Replacing images used in Rekt test resources with the images built in CI"
 
   echo "Replacing knative-eventing-test-event-library image"
-  sed -i -e "s|registry.ci.openshift.org/openshift/knative-.*:knative-eventing-test-event-library|${KNATIVE_EVENTING_TEST_EVENT_LIBRARY}|g" "$(dirname "$0")/../test/rekt/resources/eventlibrary/eventlibrary.yaml"
+  sed -i -e "s|registry.ci.openshift.org/openshift/knative-.*:knative-eventing-test-event-library|quay.io/openshift-knative/eventing/event-library:v1.6|g" "$(dirname "$0")/../test/rekt/resources/eventlibrary/eventlibrary.yaml"
```

Comment:
The env variables are used in CI to inject the dynamically built images.

Can't you define

```bash
export KNATIVE_EVENTING_TEST_EVENT_LIBRARY="quay.io/openshift-knative/eventing/event-library:v1.6"
# ... others test images env vars

source openshift/e2e-common.sh
```

/hold